Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -3827,7 +3855,41 @@ class LowerToAir {
             }
             
             if constexpr (isX86()) {
-                append(VectorSplatInt8, shiftAmount, shiftVector);
+                append(Move, shift, shiftAmount);
+                append(And32, Arg::imm(mask), shiftAmount);
+                if (value->opcode() == VectorShr && value->signMode() == SIMDSignMode::Signed && value->simdLane() == SIMDLane::i64x2) {
+                    // x86 has no SIMD 64-bit signed right shift instruction, so we scalarize it here.
+
+#if CPU(X86_64)
+                    Tmp shiftRCX = Tmp(X86Registers::ecx);
```

Comment:
IIRC, we have m_ecx for this.

Reply:
Will change it to use `m_ecx`, that definitely sounds like the right choice.

Reply:
Updated.